Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -521,22 +524,54 @@ class BocList
 
       BocList.FixHeaderSize(scrollableContainer);
       BocList.FixHeaderPosition(tableContainer, scrollableContainer);
-      BocList.FixValidationErrorOverflowSize(scrollableContainer);
       setTimeout(resizeHandler, resizeInterval);
     };
     resizeHandler();
   }
 
+  private static CreateValidationErrorOverflowResizeHandler(bocList: HTMLElement): Function
+  {
+    const tableBlock = bocList.querySelector(':scope > div.bocListTableBlock')!;
+    const tableContainer = tableBlock.querySelector<HTMLElement>(':scope > div.bocListTableContainer')!;
+    const scrollableContainer = tableContainer.querySelector<HTMLElement>(':scope > div.bocListTableScrollContainer')!;
+
+    const resizeInterval = 50;
+
+    let resizeHandler = function ()
+    {
+      if (!PageUtility.Instance.IsInDom (scrollableContainer))
+        return;
+
+      BocList.FixValidationErrorOverflowSize(scrollableContainer);
+      setTimeout(resizeHandler, resizeInterval);
+    };
+    return resizeHandler;
+  }
+
   private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
   {
-    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList
+    // minus how much space the td on the left requires to align with the error indicator column.
     // This enables sticky behaviour while still allowing for a scrollbar if required.
 
-    const width = LayoutUtility.GetWidth(scrollableContainer);
+    // -1 for rounding errors
+    const availableWidthInContainer = LayoutUtility.GetWidth(scrollableContainer) - 1;
+
+    // +1 for rounding errors
+    const scrollbarWidth = scrollableContainer.offsetWidth - scrollableContainer.clientWidth + 1;
+
+    const tdBeforeValidationErrorColumn: Nullable<HTMLElement> = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow td');
```

Comment:
Well we would just have `widthBeforeValidationFailurecolumn` as 0. We do not care about scrolling in that case, as this also means that we always have  the full length of the row.

Reply:
Check. Just for completeness sake: my assumption was, that "width" would be incorrect, subtract the whole width from the width (something like that). Anyway, not just a "it's weird" but rather an "it would be incorrect results".

Am guessing your edit also refers to "found out that there really is an actual problem"